### ISSUE TITLE
Retire slide deck for ScanPy PBMC?

### ISSUE THREAD

**Issue by mtekman** [galaxyproject/training-material#2478]
Hi, the [introduction slides with the video transcripts](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-intro/slides.html) were based on the [slide deck from the ScanPy PBMC tutorial](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/slides.html) and contain pretty much the exact same content.

Should we retire the ScanPy slide deck, and I instead make timestamped links to the introduction video? 

Or should we keep the slide deck as some kind of quick reference directly associated with the tutorial?

@nomadscientist @beatrizserrano @pavanvidem @gallardoalba

**Comment by hexylena** [comment]
`redirect_from` can be useful here if you want to redirect from the old slide deck to the new, fyi.

**Comment by mtekman** [comment]
good to know -- but I'm thinking with the video linking you're implementing, we just do away with the old deck completely.

Unless you think it will break compatibility somehow?

**Comment by hexylena** [comment]
Yeah, exactly, when you remove the old deck, adding `redirect_from` to the new directory, with the path to where the old deck used to be, can ensure that no one's links break (helps keep us FAIR ;) ) and then you're completely safe to remove the old deck.

**Comment by mtekman** [comment]
gotcha, will do!
